test/sqllogictest/select.slt

```diff
@@ -0,0 +1,21 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+mode cockroach
+
+statement ok
+CREATE TABLE foo (a integer)
+
+statement ok
+INSERT INTO foo VALUES (42)
+
+query I
+SELECT * FROM foo
+----
+42
```

Comment:
Just a note that I don't think we need this file at all

Reply:
Oops, I meant to delete this...long day. It's gone now.